Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:slim
+
+# Make container root
+USER root
+RUN mkdir /app
+
+# Install Python dependencies
+COPY requirements.txt /app
```

Comment:
For this to work, requirements.txt must be present, we will mostly be using these packages: 
```
discord.py
essentia
numpy
```